### ISSUE TITLE
some questions about over-fitting?

### ISSUE THREAD

**Issue by KKDDD** [ultralytics/yolov5#759]
## ❔Question
I used 1100 images for training to detect an object, including 700 positive samples and 400 negative samples. The accuracy on the training set reached 99%, and the recall reached 100%。
But when 200 images of the test set were used.， the accuracy of the model is only 60%, and the recall is 90%. I guess the model may be over-fitting. How to solve the problem of over-fitting?
## Additional context
I used “v5s”, and the other training parameter settings are all default values. At the beginning, the training set and the test set are the same 1100 pictures。
The figure below is the result of training
![results](https://user-images.githubusercontent.com/21326738/90397526-9077b000-e0ca-11ea-8b9f-b5999d2d7042.png)

**Comment by Ownmarc** [comment]
Your results dont look like overfitting to me. Are you sure you used the right model to do your tests ? 
Are your test images similar to the training/validation images ? If they are different in some ways, I would include them in the validation set (no test set) and use best.pt

Did you make sure there was no duplicates in training and validation ?
